Installing SwiftDiscord for iOS.md

```diff
@@ -0,0 +1,129 @@
+# Installing SwiftDiscord for iOS
+
+**Please note that you should follow the instructions in this document rather than in the main README if you intend to build for iOS.**
+
+
+## Preamble
+
+Hi there, I'm Andi. I wanted to build an iOS app that used the Discord API and SwiftDiscord fit the bill for my usecase. Unfortunately there were no official instructions at the time so I decided to try to do it  myself and document the whole process so the rest of you don't have to struggle. I'd like to thank the lead developer (nuclearace) for providing guidance throughout the whole journey; without whom I would have undoubtedly never been able to get this working. 
```

Comment:
I appreciate the preamble, but I don't think this is really need to install iOS. You could maybe shorten it and move it to the bottom in a `## Thanks`` perhaps.